AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -932,6 +934,24 @@ protected override void OnRender(DrawingContext drawingContext)
                 objectsChanged = true;
             }
 
+            if (CurrentMode == MouseMode.RectPlaceObjects && CurrentObjects.Count == 1)
+            {
+                var objectToClone = CurrentObjects[0];
```

Comment:
Just a suggestion - do we only want this to work with one object?

Couldn't we technically figure out a bounding box and related to step size and clone entire groups of objects?